code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -267,3 +267,124 @@
 			break
 	else
 		to_chat(owner, "<span class='notice'>Target is not near a camera. Cannot proceed.</span>")
+
```

Comment:
Could we add a tooltip or a help action button explaining these commands? Blue italics is what we use for OOC tips in descriptions and stuff iirc

Reply:
That is on the to do list in the main post. These to_chats are copied from the relevant actions, if that's wrong (I'll take a look) then those should be changed as well.

Reply:
Help action now added